source/services/index.js

```diff
@@ -12,7 +12,8 @@ var services = {
     pinterest:     require('./pinterest'),
     facebook:      require('./facebook'),
     twitter:       require('./twitter'),
-    gplus:         require('./gplus')
+    gplus:         require('./gplus'),
+    telegram:      require('./telegram')
```

Comment:
@vitkarpov, sure, have already thought about it. If you’d like to setup it based on the current styling and make a PR, feel free to do so :-)